app/game/Main.hs

```diff
@@ -111,7 +111,16 @@ cliParser =
   autoplay :: Parser Bool
   autoplay = switch (long "autoplay" <> short 'a' <> help "Automatically run the solution defined in the scenario, if there is one. Mutually exclusive with --run.")
   speedFactor :: Parser Int
-  speedFactor = option auto (long "speed" <> short 'm' <> value defaultInitLgTicksPerSecond <> help "Initial game speed multiplier")
+  speedFactor = option auto (long "speed" <> short 'm' <> metavar "N" <> value defaultInitLgTicksPerSecond <> help speedFactorHelp)
+  speedFactorHelp =
+    unwords
+      [ "Initial game speed."
+      , "Counted as 2^N ticks per second, default is"
+      , show defaultInitLgTicksPerSecond
+      , "meaning"
+      , show $ 2 ** fromIntegral @Int @Double defaultInitLgTicksPerSecond
+      , "t/s."
```

Comment:
Maybe also mention that a negative value of N is allowed?

Reply:
Like so?
```suggestion
      , "(Negative value is allowed, e.g. -4 means 1 tick per 8 sec.)"
```

Reply:
Sounds good, with the minor nitpick that I would say either `Negative values are allowed` or `A negative value is allowed` (personally, I prefer the first).